app/src/main/java/com/duycuong/weather/ui/screen/main/MainViewModel.java

```diff
@@ -5,14 +5,20 @@
 import android.databinding.Bindable;
 import android.location.Location;
 import android.support.v7.app.ActionBar;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports